api/elections.go

```diff
@@ -623,37 +623,37 @@ func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprout
 		Status         string         `json:"status"`
 	}{}
 	if err := json.Unmarshal(msg.Data, &body); err != nil {
-		return fmt.Errorf("cannot unmarshal request body: %w", err)
+		return ErrMarshalingServerJSONFailed.WithErr(err)
 	}
 	if len(body.OrganizationID) == 0 {
-		return fmt.Errorf("organizationId is required")
+		return fmt.Errorf("%w", ErrOrgNotFound)
```

Comment:
missed this one, should be `return ErrOrgNotFound` (yes, you can simply return the error, without `With` methods)